src/Sylius/Bundle/PaymentBundle/EventListener/PaymentMethodChangeEventListener.php

```diff
@@ -38,7 +38,7 @@ public function postUpdate(PostUpdateEventArgs $args): void
         if (array_key_exists('method', $changeSet)) {
             [$oldMethod, $newMethod] = $changeSet['method'];
 
-            if ($oldMethod !== $newMethod) {
+            if ($oldMethod !== null && $newMethod !== null && $oldMethod !== $newMethod) {
                 $this->paymentRequestCanceller->cancelPaymentRequests($entity->getId(), $newMethod->getCode());
             }
         }
```

Comment:
Is it possible to refactor those lines to use "Return Early Pattern", it will be more readable and understandable?